.github/workflows/nightly.yml

```diff
@@ -10,17 +10,21 @@ on:
         paths:
         - '.github/workflows/nightly.yml'
 
+env:
+    FORCE_COLOR: 1
 
 jobs:
 
     tests:
 
-        if: github.repository == 'aiidateam/aiida-quantumespresso'  # Prevent running the builds on forks as well
+        # Don't run scheduled tests on forks, but allow them on PRs
+        if: github.repository == 'aiidateam/aiida-quantumespresso' || github.event_name != 'schedule'
         runs-on: ubuntu-latest
 
         strategy:
+            fail-fast: true
             matrix:
-                python-version: ['3.9', '3.10', '3.11', '3.12']
```

Comment:
Not sure if you need to test all these in the nightly tests?

Reply:
I think just testing the lower/upper limit is fine indeed. 👍